composites/Plugin/Synonyms/components/SynonymsSection.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import PropTypes from "prop-types";
+import uniqueId from "lodash/uniqueId";
+import styled from "styled-components";
+import { __ } from "@wordpress/i18n";
+
+import StyledSection from "../../../../forms/StyledSection/StyledSection";
+import SynonymsInput from "./SynonymsInput";
+import HelpText from "../../Shared/components/HelpText";
+import { makeOutboundLink } from "../../../../utils/makeOutboundLink";
+
+const HelpTextLink = makeOutboundLink();
+
+const Section = styled( StyledSection )`
+	display: flex;
+	flex-direction: column;
+`;
+
+class SynonymsSection extends React.Component {
+	/**
+	 * Constructs a SynonymsSection component
+	 *
+	 * @param {Object} props          The props for this input field component.
+	 * @param {string} props.id       The id of the SynonymsField.
+	 * @param {string} props.label    The label of the SynonymsField.
+	 * @param {string} props.synonyms The initial synonyms passed to the state.
+	 * @param {string} props.onChange The change event handler.
+	 *
+	 * @returns {void}
+	 */
+	constructor( props ) {
+		super( props );
+	}
```

Comment:
Asked @atimmer for confirmation as this kind of usage of constructor seems unnecessary to me. And yes, the whole constructor could be removed in a further iteration.